### ISSUE TITLE
dist: centos not able to install artifacts

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#780]
using head:
7f4985a017d8036f848c3f7285d4b2a2bca7bcd3

build on jenkins

http://jenkins.cloudius-systems.com:8080/job/scylla-centos-rpm/47/

using RPMS build using this tag

https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/master/47/scylla.repo

not able to install them 

```
--> Processing Dependency: libboost_filesystem.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
---> Package tzdata-java.noarch 0:2015g-1.el7 will be installed
---> Package xorg-x11-fonts-Type1.noarch 0:7.5-9.el7 will be installed
--> Processing Dependency: ttmkfdir for package: xorg-x11-fonts-Type1-7.5-9.el7.noarch
--> Processing Dependency: ttmkfdir for package: xorg-x11-fonts-Type1-7.5-9.el7.noarch
--> Processing Dependency: mkfontdir for package: xorg-x11-fonts-Type1-7.5-9.el7.noarch
--> Processing Dependency: mkfontdir for package: xorg-x11-fonts-Type1-7.5-9.el7.noarch
--> Running transaction check
---> Package fontpackages-filesystem.noarch 0:1.44-8.el7 will be installed
---> Package libICE.x86_64 0:1.0.9-2.el7 will be installed
---> Package libSM.x86_64 0:1.2.2-2.el7 will be installed
---> Package libX11-common.noarch 0:1.6.3-2.el7 will be installed
---> Package libxcb.x86_64 0:1.11-4.el7 will be installed
--> Processing Dependency: libXau.so.6()(64bit) for package: libxcb-1.11-4.el7.x86_64
---> Package libxslt.x86_64 0:1.1.28-5.el7 will be installed
---> Package python-javapackages.noarch 0:3.4.1-11.el7 will be installed
--> Processing Dependency: python-lxml for package: python-javapackages-3.4.1-11.el7.noarch
---> Package qt-settings.noarch 0:19-23.5.el7.centos will be installed
--> Processing Dependency: pciutils for package: qt-settings-19-23.5.el7.centos.noarch
---> Package scylla-server.x86_64 0:development-20160112.7f4985a.el7.centos will be installed
--> Processing Dependency: libboost_unit_test_framework.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_thread.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_system.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_program_options.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_filesystem.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
---> Package ttmkfdir.x86_64 0:3.0.9-42.el7 will be installed
---> Package xorg-x11-font-utils.x86_64 1:7.5-20.el7 will be installed
--> Processing Dependency: libfontenc.so.1()(64bit) for package: 1:xorg-x11-font-utils-7.5-20.el7.x86_64
--> Processing Dependency: libXfont.so.1()(64bit) for package: 1:xorg-x11-font-utils-7.5-20.el7.x86_64
--> Running transaction check
---> Package libXau.x86_64 0:1.0.8-2.1.el7 will be installed
---> Package libXfont.x86_64 0:1.5.1-2.el7 will be installed
---> Package libfontenc.x86_64 0:1.1.2-3.el7 will be installed
---> Package pciutils.x86_64 0:3.2.1-4.el7 will be installed
---> Package python-lxml.x86_64 0:3.2.1-4.el7 will be installed
---> Package scylla-server.x86_64 0:development-20160112.7f4985a.el7.centos will be installed
--> Processing Dependency: libboost_unit_test_framework.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_thread.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_system.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_program_options.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_filesystem.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Finished Dependency Resolution
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_system.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_thread.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_unit_test_framework.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_filesystem.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_program_options.so.1.57.0()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by slivne** [comment]
Trying to install on a centos7 AWS AMI instance 

CentOS Linux 7 x86_64 HVM EBS 20150928_01-b7ee8a69-ee97-4a49-9e68-afaee216db2e-ami-69327e0c.2 (ami-61bbf104)

**Comment by syuu1228** [comment]
Need to install scylla.repo and epel-release.
Please try to copy scylla-server, scylla-jmx, scylla-tools .rpm and also scylla_install_pkg script, then run "./scylla_install_pkg -l /home/centos"

**Comment by slivne** [comment]
The issue is that we are not building any of the 3rdparty artifcast anymore - how can I force building the artifacts and not use the ones of the 3rdparty repo ?

**Comment by slivne** [comment]
The repo I provided above https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/master/47/scylla.repo 

downloads.scylladb.com/rpm/unstable/centos/master/47/3rdparty/7/noarch
downloads.scylladb.com/rpm/unstable/centos/master/47/3rdparty/7/x86_64

are all empty because the build does not build them - how can I force it to build them (and others)

**Comment by avikivity** [comment]
`./dist/redhat/build_rpm.sh -R`

**Comment by slivne** [comment]
verified it solves the issue

**Comment by vvm101** [comment]
Hi,
I removed ARBT but still have the same error.
My Linux distro is Scientific Linux release 7.2 (RH based), kernel version is:3.10.0-327.el7.x86_64

I've tried to install from 1.1 rpms directly:
sudo rpm -i scylla-server-1.1.0-20160516.dce549f.fc22.x86_64.rpm

and also tried to install adding above-mentioned scylla.repo into /etc/yum.repos.d/,
result is the same:
sudo yum install scylla-server
Loaded plugins: langpacks
epel/x86_64/metalink                                                                               | 9.3 kB  00:00:00     
epel                                                                                               | 4.3 kB  00:00:00     
repos                                                                                              | 2.6 kB  00:00:00     
scylla                                                                                             | 3.3 kB  00:00:00     
scylla-3rdparty                                                                                    | 3.3 kB  00:00:00     
scylla-3rdparty-generic                                                                            | 3.3 kB  00:00:00     
scylla-generic                                                                                     | 3.3 kB  00:00:00     
sl                                                                                                 | 3.7 kB  00:00:00     
sl-extras                                                                                          | 3.1 kB  00:00:00     
sl-fastbugs                                                                                        | 2.9 kB  00:00:00     
sl-security                                                                                        | 2.9 kB  00:00:00     
(1/6): epel/x86_64/updateinfo                                                                      | 557 kB  00:00:00     
(2/6): scylla/7/x86_64/primary_db                                                                  | 3.8 kB  00:00:00     
(3/6): scylla-3rdparty/7/x86_64/primary_db                                                         | 1.1 kB  00:00:00     
(4/6): scylla-3rdparty-generic/7/primary_db                                                        | 1.1 kB  00:00:00     
(5/6): scylla-generic/7/primary_db                                                                 | 3.2 kB  00:00:00     
(6/6): epel/x86_64/primary_db                                                                      | 4.1 MB  00:00:01     
Resolving Dependencies
--> Running transaction check
---> Package scylla-server.x86_64 0:development-20160112.7f4985a.el7.centos will be installed
--> Processing Dependency: libboost_unit_test_framework.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_thread.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_system.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_program_options.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Processing Dependency: libboost_filesystem.so.1.57.0()(64bit) for package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64
--> Finished Dependency Resolution
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_system.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_thread.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_unit_test_framework.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_filesystem.so.1.57.0()(64bit)
Error: Package: scylla-server-development-20160112.7f4985a.el7.centos.x86_64 (scylla)
           Requires: libboost_program_options.so.1.57.0()(64bit)

Could someone point me to the solution?

Thanks.
